src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -4177,7 +4177,7 @@ def _create_nw_connection_monitor_v1(cmd,
         test_configurations=None,
         test_groups=None,
         outputs=None,
-        nots=None
+        notes=None
```

Comment:
Just wonder how do you find this issue? 😄

Reply:
output from pytest's warning section